src/orga/governance.rst

```diff
@@ -309,27 +309,37 @@ Current Members of Core
 -----------------------
 In alphabetical order,
 
-* Marcel Bargull, @mbargull
-* Matthew Becker, @beckermr
-* Lori Burns, @loriab
-* Sophia Castellarin, @soapy1
-* Matt Craig, @mwcraig
+* Anthony Scopatz, @scopatz
+* Christopher J. "CJ" Wright, @CJ-Wright
+* Dougal J. Sutherland, @dougalsutherland
 * Eric Dill, @ericdill
-* Phil Elson, @pelson
 * Filipe Pires Alvarenga Fernandes, @ocefpaf
 * Isuru Fernando, @isuruf
-* Björn Grüning, @bgruening
-* Jonathan J. Helmus, @jjhelmus
 * John Kirkham, @jakirkham
+* Jonathan J. Helmus, @jjhelmus
+* Joshua Adelman, @synapticarbors
+* Lori A. Burns, @loriab
+* Marcel Bargull, @mbargull
+* Marcelo Duarte Trevisani, @marcelotrevisani
 * Marius van Niekerk, @mariusvniekerk
-* Min Ragan-Kelley, @minrk
+* Matt Craig, @mwcraig
+* Matthew R. Becker, @beckermr
 * Mike Sarahan, @msarahan
-* Anthony Scopatz, @scopatz
-* Patrick Snape, @patricksnape
+* Min Ragan-Kelley, @minrk
 * Patrick Sodré, @sodre
-* Dougal J. Sutherland, @dougalsutherland
 * Peter K. G. Williams, @pkgw
-* Christopher J. "CJ" Wright, @CJ-Wright
+* Uwe L. Korn, @xhochy
+* Wolf Vollprecht, @wolfv
```

Comment:
Isn't alphabetical order usually by last name?

Reply:
I think if the name was written as ``Surname, the first name`` I would agree with you
but I think as we wrote the name in full that might be a bit odd (at least for me) to sort it looking for the last name and writing the first name at the beginning